packages/react/src/components/Popup/Popup.tsx

```diff
@@ -438,17 +409,9 @@ export default class Popup extends AutoControlledComponent<PopupProps, PopupStat
     popupPositionClasses: string,
     rtl: boolean,
     accessibility: AccessibilityBehavior,
-    // https://popper.js.org/popper-documentation.html#Popper.scheduleUpdate
-    {
-      arrowProps,
-      placement,
-      ref,
-      scheduleUpdate,
-      style: popupPlacementStylesRaw,
-    }: PopperChildrenProps,
+    { placement, scheduleUpdate }: PopperChildrenProps,
   ) => {
     const { content: propsContent, renderContent, contentRef, mountDocument, pointing } = this.props
-    const popupPlacementStyles = _.omitBy(popupPlacementStylesRaw, _.isNaN)
```

Comment:
no need for this anymore because styles will be set directly to the DOM element by Popper js. Only valid CSS will be set and it will not leak to props. Wish I saw that PR earlier as there was no need for it